src/python/pants/backend/python/dependency_inference/rules.py

```diff
@@ -537,8 +537,8 @@ def import_rules():
         *module_mapper.rules(),
         *stripped_source_files.rules(),
         *target_types.rules(),
-        SubsystemRule(PythonInferSubsystem),
-        SubsystemRule(PythonSetup),
+        *PythonInferSubsystem.rules(),
+        *PythonSetup.rules(),
```

Comment:
Can be deleted. This is stale. We automatically register the subsystem rules in this file because they're params to rules